docs/ert/getting_started/howto/plugin_system.rst

```diff
@@ -126,6 +126,26 @@ with the documentation as specified, else respond with :code:`None`.
 When creating documentation in ERT, forward model steps will be grouped by their
 main categories (ie. the category listed before the first dot).
 
+Forward model configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Forward model steps can be configured through the plugin system by using the ``forward_model_configuration`` hook:
```

Comment:
This introduction should mention that the configuration is about specific environment variables that Ert will inject when starting the forward model.